src/components/home.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import'./index.css';
+
+
+export const Home = (props) =>{
+  return (
+    <div className = "text">
+    <h1>Pick A List</h1>
+    <div>
```

Comment:
doesn't need to be in a div

Reply:
Yes this was needed for the return to work